meta-aos-tune/recipes-aos/aos-iamanager/files/aos-iamanager.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
ditto, remove it.

Reply:
files are different, in cockpit one more dependency on aos-vis because we are using the 'visidentifier'